spec/policy/rate_limit/rate_limit_spec.lua

```diff
@@ -194,7 +194,7 @@ describe('Rate limit policy', function()
           assert(rate_limit_policy:access(context))
           assert.returns_error('limits exceeded', rate_limit_policy:access(context))
 
-          assert.equal('2', redis:get('11110_fixed_window_test3'))
+          assert.equal('1', redis:get('11110_fixed_window_test3'))
```

Comment:
I don't fully understand the purpose of this change.
Before, we increased the number of hits even when going over limits. Now we force the value to be at most equal to the limit. Why do we need to do this?

Reply:
> Before, we increased the number of hits even when going over limits.

First, we should not increase the number of hits when going over limits, because we should count the number the client calls the API backend.

The previous behavior is the below:

Case 1:
```json
fixed_window_limiters : [
  { "key" : { "name" : "a"}, "count" : 1, "window" : 10 },
  { "key" : { "name" : "b"}, "count" : 2, "window" : 10 }
]
```

After 2 API requests, 
the number of hits 'a' is '1',
the number of hits 'b' is '1'.
-> correct


Case 2:
```json
fixed_window_limiters : [
  { "key" : { "name" : "a"}, "count" : 2, "window" : 10 },
  { "key" : { "name" : "b"}, "count" : 1, "window" : 10 }
]
```

After 2 API requests,
the number of hits 'a' is '1',
the number of hits 'b' is '2'.
-> wrong